apps/website/src/components/nav-menu.tsx

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useRef, useState } from 'react'
+
+import * as NavigationMenu from '@radix-ui/react-navigation-menu'
+import { Button, Text } from '@status-im/components'
+import { DownloadIcon, ExternalIcon } from '@status-im/icons'
+import { cx } from 'class-variance-authority'
```

Comment:
Since we're using cva does it make sense to use tailwind-merge with it? 
- https://cva.style/docs/installation#handling-style-conflicts
- https://github.com/dcastil/tailwind-merge